crates/ironrdp-testsuite-core/tests/input/smoke.rs

```diff
@@ -98,24 +98,21 @@ fn smoke_keyboard() {
                         (None, None) => {}
                         (None, Some(_)) => unreachable!(),
                         (Some(pressed_packet), None) => {
-                            if let FastPathInputEvent::KeyboardEvent(flags, scancode) = pressed_packet {
+                            if let FastPathInputEvent::KeyboardEvent(flags, _scancode) = pressed_packet {
                                 ensure!(!flags.contains(KeyboardFlags::RELEASE));
-                                ensure!(scancode == u8::from(scancode))
```

Comment:
I believe the intention was to ensure that the scancode from `Operation::KeyPressed(scancode)` was the same value as the scancode from the `KeyboardEvent` PDU. Maybe a refactor went wrong, but we should keep the check, changing the second `scancode` for `other_scancode` and something like `ensure!(other_scancode == u8::from(scancode))`.